pkgs/misc/bergamot-models/models.json

```diff
@@ -0,0 +1,382 @@
+            "version": 1,
+            "API": 1,
+            "checksum": "eb9a7511ae9c89fb91ab6da1e9d5061946ad752e5801351f39c8eddca9705c74",
+            "url": "https://data.statmt.org/bergamot/models/bgen/enbg.student.tiny11.tar.gz",
+            "name": "English-Bulgarian tiny",
+            "code": "en-bg-tiny"
+        },
+        {
+            "modelName": "Polish-English tiny",
+            "shortName": "pl-en-tiny",
+            "type": "tiny",
+            "src": "Polish",
+            "trg": "English",
+            "srcTags": {
+                "pl": "Polish"
+            },
+            "trgTag": "en",
+            "repository": "Bergamot",
+            "version": 1,
+            "API": 1,
+            "checksum": "87148203cbda28421d76fffbd7d3cd6c1fc0d6dae2843c248870274d6512a388",
+            "url": "https://data.statmt.org/bergamot/models/plen/plen.student.tiny11.tar.gz",
+            "name": "Polish-English tiny",
+            "code": "pl-en-tiny"
+        },
+        {
+            "modelName": "English-Polish tiny",
+            "shortName": "en-pl-tiny",
+            "type": "tiny",
+            "src": "English",
+            "trg": "Polish",
+            "srcTags": {
+                "en": "English"
+            },
+            "trgTag": "pl",
+            "repository": "Bergamot",
+            "version": 1,
+            "API": 1,
+            "checksum": "c33219daa12e7872cf7ac8a1b86a2f3e0592ebadd7e756bf11d16d9a7725cf9b",
+            "url": "https://data.statmt.org/bergamot/models/plen/enpl.student.tiny11.tar.gz",
+            "name": "English-Polish tiny",
+            "code": "en-pl-tiny"
+        },
+        {
+            "modelName": "French-English tiny",
+            "shortName": "fr-en-tiny",
+            "type": "tiny",
+            "src": "French",
+            "trg": "English",
+            "srcTags": {
+                "fr": "French"
+            },
+            "trgTag": "en",
+            "repository": "Bergamot",
+            "version": 1,
+            "API": 1,
+            "checksum": "817a45ed9ec3228bfb797e5e14781ab7fe9f388fe1e834e280031f05089809f8",
+            "url": "https://data.statmt.org/bergamot/models/fren/fren.student.tiny11.tar.gz",
+            "name": "French-English tiny",
+            "code": "fr-en-tiny"
+        },
+        {
+            "modelName": "English-French tiny",
+            "shortName": "en-fr-tiny",
+            "type": "tiny",
+            "src": "English",
+            "trg": "French",
+            "srcTags": {
+                "en": "English"
+            },
+            "trgTag": "fr",
+            "repository": "Bergamot",
+            "version": 1,
+            "API": 1,
+            "checksum": "28deea86d2a02102a7fedf19391a7628386f01f1f532d430306a9728dc5ec2d6",
+            "url": "https://data.statmt.org/bergamot/models/fren/enfr.student.tiny11.tar.gz",
+            "name": "English-French tiny",
+            "code": "en-fr-tiny"
+        }
+    ]
+}
```

Comment:
(The "editor config" PR check fails because it wants a new line here. But it
does not make sense since this is an automatically-generated upstream JSON).

Reply:
Solved by piping to jq before writing the file.